pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,56 @@
+
+## Terminology Changes
+
+Major projects like [Python](https://www.vice.com/en_us/article/8x7akv/masterslave-terminology-was-removed-from-python-programming-language), [Go](https://www.reddit.com/r/golang/comments/gy9ylr/go_has_removed_all_uses_of_blacklistwhitelist_and/fte1zk0/), and now [Github](https://www.cnet.com/news/microsofts-github-is-removing-coding-terms-like-master-and-slave/) have all worked to eliminate the use of the problematic terms, "master" and "slave," and today Mixxx joins them.
+The issue of nomenclature and terminology in software is one example of how deeply embedded white privilege is around the globe and we hope changing these terms is one small step towards a more inclusive and diverse coding community.
+
+Although we eliminated the use of the word "slave" back in [2016](https://github.com/mixxxdj/mixxx/commit/e59916caf72a256bb28b1722759a629c5cc9cf81), "master" still appears in a number of places.
+We have a "master" code branch, a "Master Sync" feature, an "EngineMaster" object, and more.
+We are working to migrate our code away from these words, even in cases where the particular meaning of "master" was never explicitly associated with a problematic context.
+
+It’s important to note that there will be some cases where the word will remain.
+Many DJ controllers have buttons marked "master" on them, and we need to refer to those buttons by name.
+There are also the audio engineering concepts of a "master tape" or "audio mastering", so there may be areas of our documentation or code that would be less understandable if we tried to use synonyms.
+But in situations where master is used in code and previously was part of a master/slave terminology pairing, we commit to changing the language we use by DATEDATEDATE **TKTKTKTK commit to a timeline to change the main branch?? TKTKTKTKTK**.
+
+But changing language is not enough.
+
+## Diversifying Recruiting
+
+In our discussions, we quickly noted that the Mixxx contributor base is largely white, male, and either American or European.
+For some of our coders, the experience of working on Mixxx has been a doorway to employment at some of the most prestigious software companies, and we want that door to be open to anyone.
+It is our goal to make the Mixxx project as inclusive, accessible and welcoming as possible.
+As such, it is our responsibility to actively seek out and promote the participation of traditionally underserved groups.
+
+As a small volunteer-led project with no regular income, budget, or employees, we don't yet know for sure what the demographic makeup of our contributors is -- but we are aware of the research that shows that blind admission and hiring tends to result in less diverse outcomes.
+To that end we plan to sponsor an [Outreachy](https://www.outreachy.org/) intern in December 2020, the next available cycle.
```

Comment:
```suggestion
To that end we plan to participate in the [Outreachy](https://www.outreachy.org/) program and have a paid internship position by December 2020, the next available cycle.
```

"sponsor" sounds like we fund the position ourselves without the need for donations.